.pre-commit-config.yaml

```diff
@@ -24,3 +24,7 @@ repos:
           - flake8-bugbear
           - flake8-builtins
           - flake8-unused-arguments
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.6
+    hooks:
+    - id: codespell
```

Comment:
please ensure that formatting is consistent with prior records... looks totally different to me 

![image](https://github.com/datalad/datalad-installer/assets/39889/c483060c-525e-4c68-b514-0633e20a9167)

Reply:
picture didnt render, but fixed in latest commit.